src/main/groovy/org/shipkit/internal/gradle/VersionUpgrade.java

```diff
@@ -0,0 +1,84 @@
+package org.shipkit.internal.gradle;
+
+import java.io.File;
+
+/**
+ * Configuration of {@link VersionUpgradeConsumerPlugin}
```

Comment:
1. Let's start adding stuff to new package, for example "upgrades"

org.shipkit.internal.gradle.upgrades.ConsumerVersionUpgradePlugin
org.shipkit.internal.gradle.upgrades.ConsumerVersionUpgrade

2. I'm wondering if we should